### ISSUE TITLE
Use component.on('teardown', ...)

### ISSUE THREAD

**Issue by stalkerg** [TehShrike/svelte-state-renderer#4]
I think you should just change this to component.on('destroy', ...) on https://github.com/TehShrike/svelte-state-renderer/blob/master/index.js#L47 because:
```
Use component.on('destroy', ...) instead of component.on('teardown', ...) which has been deprecated and will be unsupported in Svelte 2
```

**Comment by TehShrike** [comment]
You're right!

If nobody opens a pull request, I'll try to remember to do this sometime soon

**Comment by TehShrike** [comment]
Fixed by #5.  Published as a feature bump in 1.2.0, to avoid interfering with users who are still on very early versions of Svelte.
